fdbserver/TLogServer.actor.cpp

```diff
@@ -38,7 +38,11 @@
 #include "fdbserver/LogSystem.h"
 #include "fdbserver/WaitFailure.h"
 #include "fdbserver/RecoveryState.h"
+#include "fdbserver/FDBExecArgs.h"
 #include "flow/actorcompiler.h"  // This must be the last #include.
+#if defined(CMAKE_BUILD) || !defined(WIN32)
+#include "versions.h"
+#endif
 
 using std::pair;
```

Comment:
I now realize that I made all my comments in OldTLogServer_6_0, where we also said it might be reasonable to just leave OldTLogServer_6_0 as it was.  If we're keeping the code in both places, you're welcome to share code between the two of them.  If you're not, then just pretend I left all the comments on this file instead.